### ISSUE TITLE
Keyboard focus indicator is not visible on the selected 'highlight all' control present inside the 'find' control in Windows.

### ISSUE THREAD

**Issue by msftedad** [mozilla/pdf.js#19165]
### Attach (recommended) or Link to PDF file

[compressed.tracemonkey-pldi-09.pdf (mozilla.github.io)](https://mozilla.github.io/pdf.js/web/viewer.html)

### Web browser and its version

Edge Version 131.0.2903.70 (Official build) (64-bit) Microsoft Edge is up to date.

### Operating system and its version

Windows 11 pro 24H2 OS Build: 26100.2314

### PDF.js version

1.4

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

Yes

### Steps to reproduce the problem

1. Open this PDF viewer: [compressed.tracemonkey-pldi-09.pdf (mozilla.github.io)](https://mozilla.github.io/pdf.js/web/viewer.html) in the latest Edge browser.
2. Press tab key to move to the 'Find' control and activate it using enter key.
3. Press tab key to move to the 'highlight all' control and select it using space key.
4. Verify on pressing tab key whether Keyboard focus indicator is visible on the selected 'highlight all' control present inside the 'find' control or not.

### What is the expected behavior?

On Pressing tab key, Keyboard focus indicator should be visible on the selected 'highlight all' control present inside the 'find' control in Windows.

### What went wrong?

On Pressing tab key, Keyboard focus indicator is not visible on the selected 'highlight all' control present inside the 'find' control in Windows.

### Additional context

1. Same issue observed with 'Match case', 'Match Diacritics', 'Whole Words' controls present inside the 'find' control.
2. Same issue is observed with the controls present inside 'Tools' flyout.
